src/app/common/pipes/display-account-name.pipe.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+    name: "displayAccountName",
+})
+export class DisplayAccountNamePipe implements PipeTransform {
```

Comment:
It would be nice to emphasize that this is being done specifically for Web3:

Suggested naming: `Web3EvmDisplayAccountNamePipe`

Reply:
I don't agree, this pipe may expand in the future - so you shouldn't connect it to your wallet.

Reply:
But it would be hard to extend it, since we only have a string value and no data other than that on which to base logic

What do you think about that?

Reply:
I can pass additional parameters to it

Reply:
Got it -- thanks for the explanation!

Closing the thread